frontend/src/helpers/object-to-query.ts

```diff
@@ -1,16 +1,52 @@
 export const objectToQueryString = (filters: {
-  [p: string]: string | string[];
+  [p: string]: string | string[] | Record<string, string | string[]>[];
 }): string[][] => {
   const notEmpties = Object.fromEntries(
     Object.entries(filters).filter(
-      ([_, value]) => value.length >= 1 && value !== '' && value[0] !== '',
+      ([_, value]) =>
+        value.length >= 1 &&
+        value !== '' &&
+        Array.isArray(value) &&
+        value[0] !== '',
     ),
   );
 
   return Object.entries(notEmpties).flatMap(([key, value]) => {
     if (typeof value === 'string') {
       return [[key, value]];
     }
-    return value.map((item) => [key, item]);
+    if (
+      Array.isArray(value) &&
+      (value as []).every((item) => typeof item === 'object')
+    ) {
```

Comment:
`as []`

:eye::eye:

Reply:
![image](https://user-images.githubusercontent.com/83529609/189614933-3186b928-b552-408b-98e5-a50de7c9057a.png)
Without it TS thinks that it's not an array
Although this is strange, because there is an Array.isArray(value) check